SoftLayer/CLI/virt/create.py

```diff
@@ -305,6 +327,18 @@ def cli(env, **args):
 def _validate_args(env, args):
     """Raises an ArgumentError if the given arguments are not valid."""
 
+    if all([args['cpu'], args['flavor']]):
+        raise exceptions.ArgumentError(
+            '[-c | --cpu] not allowed with [-f | --flavor]')
+
+    if all([args['memory'], args['flavor']]):
+        raise exceptions.ArgumentError(
+            '[-m | --memory] not allowed with [-f | --flavor]')
+
+    if all([args['dedicated'], args['flavor']]):
```

Comment:
You could also include one for `host-id` and `flavor` right?

Reply:
Yep. Adding.